docker/bootstrap

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+set -e
+
+cd "$(dirname "$0")/.."
+
+error_msg() { printf "\033[31m%s\033[0m\n" "$*"; }
+notice_msg() { printf "\033[33m%s\033[0m " "$*"; }
+success_msg() { printf "\033[32m%s\033[0m\n" "$*"; }
+
+notice_msg "Checking dependencies..."
+if ! which docker &>/dev/null; then
```

Comment:
I'd use `if ! command -v docker >/dev/null 2>&1; then` here - a bit more verbose, but more portable given we're calling this script with `/bin/sh`.